adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    list_letter_pool =[]
+    for letter, number in LETTER_POOL.items():
+        letter_quantity = letter * number
+        list_letter_pool.append(letter_quantity)
+
+
+    letter_string = ''.join(list_letter_pool)
+   
+    # Available letters with frequencies list
+    available_letters = list(letter_string)
+    
+    hand = []
+
+# Returns a list of 10 random letters
+    while len(hand) != 10:
+        random_letter = random.choice(available_letters)
+        hand.append(random_letter)
+        for letter in available_letters:
+            if letter in hand:
+                available_letters.remove(letter)
```

Comment:
Your logic ends up being a little bit buggy! I noticed this because when I ran your tests (I run them like 50 times as I grade lol), sometimes 5 tests failed, sometimes 6 tests failed for me.

I spent some time trying to learn about the bug and this is what I found. While `len(hand) != 10`, you want to:
    - pick a random letter (`random_letter`)
    - check if that random letter is available (aka in `available_letters`)
    - Only if that random letter is available, then do we add it to the hand and remove it from `available_letters`

Your code currently does something else. While `len(hand) != 10`, it:
    - always adds the `random_letter` to the hand
    - Loops through all `available_letters`
    - If the available letter is in the hand, then it remove it from `available_letters`

The key difference is when you add to `hand` and when you remove from `available_letters`

This is a fun bug to fix and an interesting refactoring of your code! Let me know what questions come up.